packages/terra-core-docs/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Changed
+  * Updated all paginator examples for `terra-paginator` to include passed in ARIA contexts.
```

Comment:
nit for clarity
```suggestion
  * Updated all paginator examples for `terra-paginator` to use new props.
```

Reply:
https://github.com/cerner/terra-core/pull/3867/commits/450ee57249ee65b6a8508c71973f7b2373d577ae#diff-0d87855fd9582c564e6ca6a9654fcfa4099b7e717a0704a990d80316f05bdf59R6